template/deployments/helm/charts/production.yaml

```diff
@@ -1,6 +1,6 @@
 image:
   repository: "{{APPNAME}}"
-  pullPolicy: IfNotPresent
```

Comment:
should this be switched for helm and kube manifests too?

Reply:
yes, i see we didn't even have pullpolicies on those, adding them now